.github/ISSUE_TEMPLATE/b-bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Report a bug
+description: You encountered a bug in the geolocator plugin that you would like to report.
```

Can we describe a bit more detailed what is considered a "bug"? How about something like this:
```suggestion
description: You encountered a bug in the geolocator plugin resulting in application crashes, compilation errors or invalid/ unexpected results.
```